miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -595,6 +595,67 @@ var Utils = Utils
         // Zero-pad month and day if necessary
         return yyyy + "-" + (mm < 10 ? "0" + mm : mm) + "-" + (dd < 10 ? "0" + dd : dd);
       },
+      createBoxDialog: function(callback){
+        var boxFields = [{
+          property: 'alias',
+          type: 'text',
+          label: 'Alias',
+          value: '',
+          required: true
+        }, {
+          property: 'use',
+          type: 'select',
+          label: 'Box Use',
+          values: Constants.boxUses,
+          getLabel: Utils.array.getAlias,
+          required: true
+        }, {
+          property: 'size',
+          type: 'select',
+          label: 'Box Size',
+          values: Constants.boxSizes,
+          getLabel: function(boxSize){
+            return boxSize.rowsByColumns;
```

Comment:
Should also display "scannable" if the size is scannable, like on the Edit Box page